arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -3297,6 +3303,7 @@
 				4CAC91D72242AAB700663A10 /* Load WFS with XML query */,
 				12D6602B226697040005DE8A /* Group layers */,
 				4CE15D4822F37968005F2968 /* Control annotation sublayer visibility */,
+				F1EF67DB27E2AE200049FA9A /* Map image layer tables */,
```

Comment:
We tend to put latest samples at the top of each folder. Until we make the sort-alphabetically change, let's still keep that.